force-app/main/default/classes/SOQL.cls

```diff
@@ -314,18 +367,25 @@ public with sharing class SOQL {
      */
     public static void cloneLineItems(Id origOppId, List<Id> cloneOppIds) {
         // Query the original Opportunity Line Items
-
+        List<OpportunityLineItem> oldOppLineItems = [SELECT Id, PricebookEntryId, UnitPrice FROM OpportunityLineItem WHERE OpportunityId = :origOppId];
         // Create a list to hold the new line items
-
+        List<OpportunityLineItem> newOppLineItems = new List<OpportunityLineItem>();
         // Generate a random number between 1 and 3
-
+        Integer randomNumber = Integer.valueof((Math.random() * 3));
```

Comment:
Chances are this is only generating a random number between 0 and 2. Here's the general formula:
`Integer randomNumber = (Integer) (Math.random() * (max - min + 1)) + min`

In your case, you seem to be missing the + 1 on the end (this would bump the 0 to 1 and the 2 to 3 to give you numbers between 1 and 3.